docs/netdata-cloud-onprem/getting-started-light-poc.md

```diff
@@ -13,16 +13,25 @@ To install the whole environment, log in to the designated host and run:
 ```shell
 curl https://netdata-cloud-netdata-static-content.s3.amazonaws.com/provision.sh -o provision.sh
 chmod +x provision.sh
-sudo ./provision.sh --install
+sudo ./provision.sh install \
+      -key-id "" \
+      -access-key "" \
+      -onprem-license-key "" \
+      -onprem-license-subject "" \
+      -onprem-url "" \
+      -certificate-path "" \
+      -private-key-path ""
 ```
 
 What does the script do during installation?
-1. Prompts user to provide:
-   - ID and KEY for accessing the AWS (to pull helm charts and container images)
-   - License Key
-   - URL under which Netdata Cloud Onprem PoC is going to function (without protocol like `https://`)
-   - Path for certificate file (PEM format)
-   - Path for private key file (PEM format)
+1. Prompts for user to provide:
```

Comment:
is this expected formatting?

<img width="1332" alt="Screenshot 2024-01-19 at 12 42 11" src="https://github.com/netdata/netdata/assets/22274335/e1de11d6-203f-4eab-a7d5-5ad624489faf">

Reply:
it does not render this way on my computer, lemme tweak it. Nice catch, thanks